src/storage/src/hummock/sstable/block.rs

```diff
@@ -473,6 +552,7 @@ impl BlockBuilder {
             let offset = self.buf.len() as u32;
 
             self.restart_points.push(offset);
+            self.epoch_group.push(Vec::default());
```

Comment:
We can use `Vec::with_capacity` since we know the number of epochs per group beforehand.

Reply:
we cant not know the number of key per restart_point due to the Flexible length encoding, i will add some document